### ISSUE TITLE
EDCarnage errors on open

### ISSUE THREAD

**Issue by Agnos1** [mmomtchev/EDCarnage#3]
Tried updating Java as it's a Java Script issue
![Screenshot (54)](https://user-images.githubusercontent.com/103075675/161853536-fdd011bf-c109-478b-833d-607e819d3b4e.png)

**Comment by mmomtchev** [comment]
Do you get it every time you launch?

**Comment by mmomtchev** [comment]
Normally this would be indicative of a Journal file that `EDCarnage` fails to read - try copying your journal files to another folder, then delete the original folder and rename the new folder to the old name - if this fixes it, you should probably start considering replacing your storage device

**Comment by Agnos1** [comment]
Worked fine on the old version. Happens every time on launch even after uninstalling the ver 1.00


Sent from Yahoo Mail for iPhone


On Wednesday, April 6, 2022, 11:58 AM, Momtchil Momtchev ***@***.***> wrote:




Do you get it every time you launch?

**Comment by Agnos1** [comment]
Do you mean the Elite dangerous journal folder?


Sent from Yahoo Mail for iPhone


On Wednesday, April 6, 2022, 12:16 PM, Momtchil Momtchev ***@***.***> wrote:




Normally this would be indicative of a Journal file that EDCarnage fails to read - try copying your journal files to another folder, then delete the original folder and rename the new folder to the old name - if this fixes it, you should probably start considering replacing your storage device

**Comment by mmomtchev** [comment]
Yes, the journals folder

**Comment by Agnos1** [comment]
There are a number of journal files within the "elite dangerous" folder including some others like backpack, cargo, module, etc. Should I delete the entire ED file after copying to a new file and renaming the new file to the old name? And if it fixes it, why would this mean that my brand new storage device on a less than 6 months old computer need replacing?
Carl

    On Wednesday, April 6, 2022, 12:16:32 PM PDT, Momtchil Momtchev ***@***.***> wrote:  
 
 


Normally this would be indicative of a Journal file that EDCarnage fails to read - try copying your journal files to another folder, then delete the original folder and rename the new folder to the old name - if this fixes it, you should probably start considering replacing your storage device

**Comment by mmomtchev** [comment]
You need to remove only the `Journal.*` files

**Comment by Agnos1** [comment]
So it’s ok to delete the other non-journal files?


Sent from Yahoo Mail for iPhone


On Sunday, April 10, 2022, 10:30 AM, Momtchil Momtchev ***@***.***> wrote:




You need to remove only the Journal.* files

**Comment by mmomtchev** [comment]
Just move them somewhere else - if you delete them, you will lose your history - all ED tools requires those files

**Comment by Agnos1** [comment]
So we want a new folder that’s blank in order to test Carnage, while saving all the original files in a new folder. Am I interpreting correctly?


Sent from Yahoo Mail for iPhone


On Sunday, April 10, 2022, 11:01 AM, Momtchil Momtchev ***@***.***> wrote:




Just move them somewhere else - if you delete them, you will lose your history - all ED tools requires those files

**Comment by mmomtchev** [comment]
Yes, then start a new session to check if it works - it won't work without any journal files

**Comment by Agnos1** [comment]
I must be obtuse, but copying all the files to a new folder, deleting the old, renaming the new one containing all the same files seems to me I’d end up with an exact copy of the the original journal file and I don’t see why the app would suddenly work. 

Sent from Yahoo Mail for iPhone


On Sunday, April 10, 2022, 11:07 AM, Momtchil Momtchev ***@***.***> wrote:




Yes, then start a new session to check if it works - it won't work without any journal files

**Comment by mmomtchev** [comment]
yes

**Comment by Agnos1** [comment]
Another way to approach this question is intent- what are we trying to get EDCarnage to ignore or consider re this Elite dangerous file? 


Sent from Yahoo Mail for iPhone


On Sunday, April 10, 2022, 11:12 AM, Momtchil Momtchev ***@***.***> wrote:




yes

**Comment by Agnos1** [comment]
I tried copying the journal.* to another folder and it couldn't read one of the files. I've pinpointed the journal file in question. What do you suggest I do, delete it?

    On Wednesday, April 6, 2022, 12:16:32 PM PDT, Momtchil Momtchev ***@***.***> wrote:  
 
 


Normally this would be indicative of a Journal file that EDCarnage fails to read - try copying your journal files to another folder, then delete the original folder and rename the new folder to the old name - if this fixes it, you should probably start considering replacing your storage device

**Comment by Agnos1** [comment]
I went ahead and followed the steps. It is now stable and working fine. I think I’ll leave things well enough alone at this point!
Thanks for your help!
Carl


Sent from Yahoo Mail for iPhone


On Sunday, April 10, 2022, 5:10 PM, Carl Johnson ***@***.***> wrote:

 I tried copying the journal.* to another folder and it couldn't read one of the files. I've pinpointed the journal file in question. What do you suggest I do, delete it?

    On Wednesday, April 6, 2022, 12:16:32 PM PDT, Momtchil Momtchev ***@***.***> wrote:  
 
 


Normally this would be indicative of a Journal file that EDCarnage fails to read - try copying your journal files to another folder, then delete the original folder and rename the new folder to the old name - if this fixes it, you should probably start considering replacing your storage device

**Comment by Agnos1** [comment]
One thing, though, is that the Session section at the top which includes Kills and Bounties total and Last Hour, all the numbers remain at zero throughout the mission. Not super important as the Faction info, up to date works fine now. But it would be good to know if this can be fixed.
Thanks again!
Carl


    On Sunday, April 10, 2022, 07:25:40 PM PDT, Carl Johnson ***@***.***> wrote:  
 
 I went ahead and followed the steps. It is now stable and working fine. I think I’ll leave things well enough alone at this point!
Thanks for your help!
Carl


Sent from Yahoo Mail for iPhone


On Sunday, April 10, 2022, 5:10 PM, Carl Johnson ***@***.***> wrote:

 I tried copying the journal.* to another folder and it couldn't read one of the files. I've pinpointed the journal file in question. What do you suggest I do, delete it?

    On Wednesday, April 6, 2022, 12:16:32 PM PDT, Momtchil Momtchev ***@***.***> wrote:  
 
 


Normally this would be indicative of a Journal file that EDCarnage fails to read - try copying your journal files to another folder, then delete the original folder and rename the new folder to the old name - if this fixes it, you should probably start considering replacing your storage device
